### PR TITLE
fix(pageheader): replace dropdown with Menu component in Pageheader stories

### PR DESCRIPTION
**Pull request by youngheart12**
…ropdown with Select component

### What does this implement/fix? Explain your changes.
Replaces Dropdown with Select component in PageHeader
...

### Does this close any currently open issues?
#2389
...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Comment by youngheart12** [comment]
@anuradha9712 Please review

File: core/components/organisms/pageHeader/__stories__/Responsiveness.story.jsx [hunk 1]
```diff
@@ -142,13 +142,18 @@ const customCode = `() => {
...
+          <Menu.Item className="d-flex align-items-center">
+            <Icon name="print" className="mr-4 my-2" />
+            <Text>Edit</Text>
+          </Menu.Item>
+          <Menu.Item className="d-flex align-items-center">
+            <Icon name="assignment_turned_in" className="mr-4 my-2" />
+            <Text>Complete</Text>
+          </Menu.Item>
+        </Menu.List>
+      </Menu>
```
Comment (anuradha9712) [inline core/components/organisms/pageHeader/__stories__/Responsiveness.story.jsx]: Please fix the indentation of the selected code. `<Menu.List>` is the children of `<Menu>` so it should be indented inside the `Menu`

Comment (youngheart12) [reply core/components/organisms/pageHeader/__stories__/Responsiveness.story.jsx]: @anuradha9712 Updated

**Comment by anuradha9712** [comment]
Please update the commit message to "fix(pageheader): replace dropdown with Menu component in Pageheader stories"

**Comment by youngheart12** [comment]
@anuradha9712 Updated

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**